catroid/src/main/res/layout/fragment_add_new_user_brick.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Catroid: An on-device visual programming system for Android devices
+  ~ Copyright (C) 2010-2018 The Catrobat Team
+  ~ (<http://developer.catrobat.org/credits>)
+  ~
+  ~ This program is free software: you can redistribute it and/or modify
+  ~ it under the terms of the GNU Affero General Public License as
+  ~ published by the Free Software Foundation, either version 3 of the
+  ~ License, or (at your option) any later version.
+  ~
+  ~ An additional term exception under section 7 of the GNU Affero
+  ~ General Public License, version 3, is available at
+  ~ http://developer.catrobat.org/license_additional_term
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  ~ GNU Affero General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Affero General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+
+<!-- Overdraw lint warning is ignored - fragments need refactoring -->
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/app_background"
+    android:orientation="vertical"
+    tools:ignore="Overdraw">
+
+    <org.catrobat.catroid.ui.BrickLayout
+        android:id="@+id/brick_user_brick"
+        style="@style/BrickContainer.Look.Medium"
+        android:layout_marginTop="10dp"
+        android:background="@drawable/brick_darkblue_1h">
+
+        <include layout="@layout/icon_brick_category_look" />
+
+        <TextView
+            style="@style/BrickText.SingleLine"
+            android:labelFor="@+id/brick_user_brick_edit_text"
+            android:text="@string/brick_user_defined">
```

Comment:
```suggestion
            android:labelFor="@+id/brick_user_brick_edit_text">
```

Or use android:text="", i.e., an empty string if necessary. 
*Reason:* The blue brick should initially be empty. No text shall be shown there.